state/factory/statedb.go

```diff
@@ -66,7 +65,7 @@ func PrecreatedStateDBOption(kv db.KVStore) StateDBOption {
 
 // DefaultStateDBOption creates default state db from config
 func DefaultStateDBOption() StateDBOption {
-	return func(sdb *stateDB, cfg config.Config) error {
+	return func(sdb *stateDB, cfg Config) error {
```

Comment:
cfg *Config

Reply:
ok, updated. But Struct better in here, because pointer can be nil, cfg can not be nil